src/backend/metal/src/command.rs

```diff
@@ -599,6 +599,17 @@ impl RawCommandBuffer<Backend> for CommandBuffer {
                                         }
                                     }
                                 }
+                                ConstantBuffer(ref buffers) => {
+                                    if let EncoderState::Render(ref encoder) = inner.encoder_state {
+                                        for (i, ref bref) in buffers.iter().enumerate() {
+                                            if let Some(ref buffer) = **bref {
+                                                let offset = 0; // TODO: ?
```

Comment:
should be stored in the `DescriptorSetBinding` alongside the buffer ID

Reply:
Oh right, in the ConstantBuffer variant itself? Alright then

Reply:
Basically, store it besides the buffer. Buffer + offset is like a constant buffer view in D3D12